contracts/factories/WrappedExternalBribeFactory.sol

```diff
@@ -21,4 +18,10 @@ contract WrappedExternalBribeFactory {
         oldBribeToNew[existing_bribe] = last_bribe;
         return last_bribe;
     }
+
+    function setVoter(address _voter) external {
+        require(!_initialized, "Already initialized");
```

Comment:
what if we have made a mistake setting a wrong voter address?
current code will not allow further changing.